### DISCUSSION TITLE
Is there any way to add in new phase (reindex) in MTEX?

### DISCUSSION THREAD

**Discussion by trudator** [mtex-toolbox/mtex#1624] [Ask Anything] [answered]
Hi folks, I've done scanning using EDAX software but didn't pick an HCP phase at the beginning of the scan. Is that possible to reindex/recompute with a new phase using MTEX (other solutions or software are also welcomed)?

Thanks.

**Comment by kilir** [comment, marked as answer]
Hi,
no, there's no pattern indexing in mtex. Maybe Kikuchipy would be an option if you still have the pattern
Cheers,
Rüdiger

**Reply by trudator** [reply]
Hi Rüdiger,

Thanks for your answer. Just clarification: can I know what is the extension of a pattern file?

**Reply by hakonanes** [reply]
> can I know what is the extension of a pattern file?

The most common vendor fortmats that I know of are `.bcf` or an HDF5 file (Bruker Nano), `.up1`, `.up2` or an HDF5 file (EDAX TSL) and `.ebsp` or `.h5oina` (Oxford Instruments). All but `.bcf` can be read in Python/kikuchipy (https://kikuchipy.org/en/stable/tutorials/load_save_data.html#Supported-file-formats).

**Reply by trudator** [reply]
Hi @hakonanes, Thanks for your info. Is kikuchipy able to read and reindex with added phases? I know that kikuchipy can read HDF5 of EDAX but not sure if it's able to rephase and export new .ang or .osc files? 

Thanks in advance.

**Reply by hakonanes** [reply]
There is unfortunately no established workflow for re-indexing, I recommend checking the tutorials (https://kikuchipy.org/en/stable/tutorials/index.html) for details.

**Reply by trudator** [reply]
Thanks for your recommendation.

**Reply by hakonanes** [reply]
@trudator Note that kikuchipy v0.9 can now, via PyEBSDIndex v0.2, perform Hough indexing of all Laue classes (including HCP or 6/mmm).

**Comment by lfgmorales** [comment]
Hi

No, the only option you have is that…assuming you save the patterns or the hough, you rescan with OIM

Cheers

L

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 3 Nov 2022, at 18:43, trudator ***@***.***> wrote:

﻿

Hi folks, I've done scanning using EDAX software but didn't pick an HCP phase at the beginning of the scan. Is that possible to reindex/recompute with a new phase using MTEX (other solutions or software are also welcomed)?

Thanks.

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1624>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN5JAYRNRSIO4PAGZK3WGP2UBANCNFSM6AAAAAARWMOE6Y>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1624", "url": "https://github.com/mtex-toolbox/mtex/discussions/1624", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Comment by lfgmorales** [comment]
Hi

It can be *up2 file if you saved as streaming files


Cheers


L

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 3 Nov 2022, at 21:01, trudator ***@***.***> wrote:

﻿

Hi Rüdiger,

Thanks for your answer. Just clarification: can I know what is the extension of a pattern file?

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1624#discussioncomment-4050795>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN45NHVXJ7BUGJ2JSHDWGQK2PANCNFSM6AAAAAARWMOE6Y>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1624#discussioncomment-4050795", "url": "https://github.com/mtex-toolbox/mtex/discussions/1624#discussioncomment-4050795", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Comment by trudator** [comment]
Thanks you guys!
